VideoIndexerEnabledByArc/scripts/install_aks_cluster.sh

```diff
@@ -0,0 +1,186 @@
+workerVmSize="Standard_D32a_v4" # 32 vcpus, 128 GB RAM
+tags="createdBy=vi-arc-extension"
+#=========Install CLI Tools =====================#
+install_cli_tools
+
+#===========================================================================================================#
+# @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ Deploy Infra @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+#===========================================================================================================#
+echo "================================================================"
+echo "========== Deploying new AKS and ARC Enabled Resources ========="
+echo "================================================================"
+
+echo "Deploying Resources: [Resource group: $rg, AKS: $aks, Connected-Cluster Name: ${connectedClusterName}]"
+echo "create Resource group"
+az group create --name $rg --location $region --output table --tags $tags
+
+#=======================================================================#
+#======== Create AKS Cluster( Simulates User On Prem Infra) ============#
+#=======================================================================#
+echo -e "\t create aks cluster Name: $aks , Resource Group $rg- ***start***"
+az aks create -n $aks -g $rg \
+    --enable-managed-identity\
+    --kubernetes-version ${aksVersion} \
+    --enable-oidc-issuer \
+    --nodepool-name system \
+    --os-sku AzureLinux \
+    --node-count 2 \
+    --tier standard \
+    --generate-ssh-keys \
+    --network-plugin kubenet \
+    --tags $tags \
+    --node-resource-group $nodePoolRg \
+    --node-vm-size $nodeVmSize \
+    --node-os-upgrade-channel NodeImage --auto-upgrade-channel node-image
+
+echo -e "\t create aks cluster Name: $aks , Resource Group $rg- ***done***"
+echo "Adding another worload node pool"
+az aks nodepool add -g $rg --cluster-name $aks  -n workload \
+        --os-sku AzureLinux \
+        --mode User \
+        --node-vm-size $workerVmSize \
+        --node-osdisk-size 100 \
+        --node-count 0 \
+        --max-count 10 \
+        --min-count 0  \
+        --tags $tags \
+        --enable-cluster-autoscaler \
+        --max-pods 110
+echo "Adding another workload node pool ***done***"
+#=============================================#
+#============== AKS Credentials ==============#
+#=============================================#
+echo -e  "\tConnecting to AKS and getting credentials  - ***start***"
+az aks get-credentials --resource-group $rg --name $aks --admin --overwrite-existing
+echo "AKS connectivity Sanity test"
+kubectl get nodes
+echo -e "\tconnect aks cluster - ***done***"
+#=============================================#
+#============== Add ingress controller =======#
+#=============================================#
+echo -e "\tAdding ingress controller -- ***start***"
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.9.4/deploy/static/provider/cloud/deploy.yaml
```

Comment:
I think 1.9.5 is the latest

Reply:
moved to using the latest from main branch